api/http/server.go

```diff
@@ -121,3 +123,20 @@ func (server *Server) Start() error {
 	}
 	return http.ListenAndServe(server.BindAddress, server.Handler)
 }
+
+// GetHealth checks the response of GET /api/status and exits
+func GetHealth(addr string) {
+	addr = "http://" + addr + "/api/status"
```

Comment:
I think that the only parameter required for this function should be the port number. It'll always check for localhost inside the container or running without Docker.

Also, this methods should return either 0 (success) or 1 (failure) if code the status is not 200 as well as the error message if there is an error. The os.Exit part should be integrated in the `main.go` rather than in the `http` package.

Reply:
Flags don't provide the port number explicitly, but "Address and port to serve Portainer". Therefore, `addr` is used raw, to avoid parsing/splitting the string.

Reply:
Agree, we probably need to change this. Note that this might cause troubles when associated with "localhost".